ruby/red-arrow/ext/arrow/raw-records.cpp

```diff
@@ -306,6 +306,10 @@ namespace red_arrow {
 
   VALUE
   record_batch_each_raw_record(VALUE rb_record_batch){
+    if (!rb_block_given_p()) {
+      return rb_funcall(rb_record_batch, rb_intern("to_enum"), 1, ID2SYM(rb_intern("each_raw_record")));
```

Comment:
If there is a good way to return a Enumerator object, could you teach me?🙏

Reply:
fix: [f59e5ff](https://github.com/apache/arrow/pull/37600/commits/f59e5ff942fda34fd7c7ed1b5edce7ace0732020)
Thanks. I tried to use `RETURN_SIZED_ENUMERATOR()`🙏